app/src/main/java/org/mozilla/fenix/components/Analytics.kt

```diff
@@ -28,14 +29,23 @@ class Analytics(
     private val context: Context
 ) {
     val crashReporter: CrashReporter by lazy {
-        val sentryService = SentryService(
-            context,
-            BuildConfig.SENTRY_TOKEN,
-            tags = mapOf("geckoview" to "$MOZ_APP_VERSION-$MOZ_APP_BUILDID"),
-            sendEventForNativeCrashes = true
-        )
+        var services = listOf<CrashReporterService>()
+        val crashReportingEnabled = BuildConfig.CRASH_REPORTING
+        val isReleaseBuild = BuildConfig.BUILD_TYPE == "release"
+
+        if (crashReportingEnabled && isReleaseBuild) {
```

Comment:
Since we can't have both services I would just wrap this in a `!BuildConfig.SENTRY_TOKEN.isNullOrEmpty()` and also use the same pattern for adding to the list `services += sentryService`